src/services/BiometricService.ts

```diff
@@ -0,0 +1,47 @@
+import ReactNativeBiometrics from 'react-native-biometrics';
+
+type BiometryType =
+  | undefined
+  | ReactNativeBiometrics.FaceID
+  | ReactNativeBiometrics.TouchID
+  | ReactNativeBiometrics.Biometrics;
+
+export default class BiometricService {
+  static FaceID = ReactNativeBiometrics.FaceID;
+  static TouchID = ReactNativeBiometrics.TouchID;
+  static Biometrics = ReactNativeBiometrics.Biometrics;
+
+  constructor() {
+    this.getBiometricsAvailability();
+  }
+
+  biometryType: BiometryType;
+
+  getBiometricsAvailability = async () => {
```

Comment:
usually getters `getBiometricsAvailability` return something. This one returns always undefined. Just naming issue ;)